tests/integration.rs

```diff
@@ -223,7 +223,13 @@ impl Drop for ProtocolTester {
     fn drop(&mut self) {
         self.tcp_device.process.kill().unwrap();
         self.unix_device.process.kill().unwrap();
-        fs::remove_file("test_chain_id_priv_validator_state.json").unwrap();
+
+        match fs::remove_file("test_chain_id_priv_validator_state.json") {
```

Comment:
It might be better to make this configurable from the outside (currently it's always "chain_id" + "priv_validator_state.json") and store it in a constant (per test?).